tests/qxmppmixinvitation/tst_qxmppmixinvitation.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2008-2020 The QXmpp developers
```

Comment:
```suggestion
 * Copyright (C) 2008-2021 The QXmpp developers
```
(I just updated the copyright headers)